syntaxes/gdb.tmLanguage.json

```diff
@@ -86,6 +135,14 @@
 
 					}
 				},
+				{
+					"name": "meta.command.delete.gdb",
+					"match": "^\\s*(d(?:el(?:e(?:t(?:e)?)?)?)?)(?:\\s+b(?:r(?:e(?:a(?:k(?:p(?:o(?:i(?:n(?:t(?:s)?)?)?)?)?)?)?)?)?)?)?(?:\\s+((?:(?:\\d+|\\$\\w+)\\s*)+))?\\s*$",
+					"captures": {
+						"1": { "name": "keyword.control.command.gdb" },
+						"2": { "name": "entity.name.tag.breakpoint.gdb" }
+					}
+				},
```

Comment:
Why moving delete ?

Reply:
Hm, I think to move all breakpoint related commands together and have the same set in same order for `break` and `skip` - should I revert that part or leave as-is?

Reply:
leave it, acceptable reason.